### ISSUE TITLE
Suggestion: Add more comments to code

### ISSUE THREAD

**Issue by RadCraftplay** [LogicAndTrick/sledge#243]
If you add more comments to code, it will be easier to read and understand. More developers and more often would help you developing Sledge and open more pull requests

**Comment by LogicAndTrick** [comment]
Yeah, I definitely want to get to that point. I'm hoping that after the big re-architecture I'm working on is finished, everything will be much more modular and easier to understand. Then I will be adding comments where I can and I also want to write some developer documentation on how to do various things as well.
